docs/user_guide/architecture.rst

```diff
@@ -0,0 +1,128 @@
+.. _architecture:
+
+Testbenches Architecture
+===============================================================================
+
+File structure of a project
+-------------------------------------------------------------------------------
+
+.. tip::
+
+   In ``base/`` you can find a test bench base design.
+
+Project files for test benches
```

Comment:
Redo the structure as it was decided the last time. Descriptions can be added while we review the documentation